app/README.md

```diff
@@ -58,34 +58,73 @@ Note: make sure TypeScript and Javascript Language Features are enabled in VS Co
 - `incremental: true` -- enables incremental typechecking. Incremental typechecking creates a series of `.tsbuildinfo` files to dave information from last compilation. This expedites type checking during build.
 - `baseUrl: "."` & `paths: { @pages/*: [pages/*] }` -- These two, in tandem, setup module path aliases for cleaner imports. To utilize this, import files like: `import Home from "@pages/index";`
 
-### Package.json
+## Package.json
 
-#### Scripts
+### Scripts
 
 These can be run with `npm run [name of script]`.
 
+- `build`: builds the app
+- `dev`: runs the app in development mode
 - `format`: instructs prettier to rewrite files with fixes for formatting violations.
 - `format-check`: instructs prettier to only check files for violations without fixing them.
+- `lint`: runs ESLint and reports warnings and errors
+- `postinstall`: copies the USWDS static assets into the project
```

Comment:
It could be worth noting that postinstall also runs automagically with the `install` command (https://docs.npmjs.com/cli/v8/using-npm/scripts) although maybe that is obvious 😅

Reply:
Maybe obvious to some, but not to me, good call!